runtime/ms-rest-azure/lib/cloudError.js

```diff
@@ -108,6 +114,20 @@ class CloudError extends Error {
             type: {
               name: 'Object'
             }
+          },
+          additionalInfo: {
+            required: false,
+            serializedName: 'additionalInfo',
+            type: {
+              name: 'Sequence',
+              element: {
+                required: false,
+                serializedName: 'AdditionalInfoElementType',
+                type: {
+                  name: 'Object'
+                }
+              }
+            }
```

Comment:
How do we expect end-users to use this information? Do we really expect them to just output it to log, such as `console.log(JSON.stringify(cloudError.additionalInfo))`? If you expect them to perform any type of logic/decisions on this data, then I'd recommend coming up with a better type definition than just Object. If you just expect `additionalInfo` to be output to a log, then this is probably fine.

Reply:
Defined 'type' and 'info' property as amarzavery suggested. I expect the users to usually log these additional error information. Also 'type' property will tell the error type which I think can be used for any further decision.